src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.models.Task;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+/**
+ * Removes Task from the task list
+ */
+public class DeleteCommand extends Command {
+    private final int index;
```

Comment:
Perhaps constant names should be all uppercase?
```suggestion
    private final int INDEX;
```